UITests/TransferMethod/SelectTransferMethodTypeTests.swift

```diff
@@ -102,4 +105,35 @@ class SelectTransferMethodTypeTests: BaseTests {
         XCTAssertTrue(app.tables["addTransferMethodTable"].exists)
         XCTAssertTrue(app.tables.staticTexts["Account Information - United States (USD)"].exists)
     }
+
+    func testSelectTransferMethodType_verifyCountrySelectionSearch() {
+        selectTransferMethodType.tapCountry()
+
+        selectTransferMethodType.typeSearch(input: "Canada")
+        XCTAssertEqual(app.tables.cells.count, 1)
```

Comment:
Please add an additional check so that when you click Canada, the values in the next page get will updated as well.

Country: Canada
Currency: CAD